.github/workflows/tests_arm64.yaml

```diff
@@ -5,4 +5,8 @@ jobs:
   test-linux-arm64:
     uses: ./.github/workflows/tests-template.yml
     with:
-      runs-on: "['actuated-arm64-4cpu-16gb']"
+      runs-on: actuated-arm64-4cpu-16gb
```

Comment:
Nice work on this.

Suggest `linux-unit-test-4-cpu-race` runs on `actuated-arm64-8cpu-32gb` as those jobs can take around 30 minutes + which we want to try reduce if possible.

Reply:
Done